utils/config.go

```diff
@@ -353,7 +344,6 @@ func GenerateClientConfig(c *model.Config, diagnosticId string, license *model.L
 	props["BuildEnterpriseReady"] = model.BuildEnterpriseReady
 
 	props["SiteURL"] = strings.TrimRight(*c.ServiceSettings.SiteURL, "/")
-	props["WebsocketURL"] = strings.TrimRight(*c.ServiceSettings.WebsocketURL, "/")
```

Comment:
Why is this being removed? Perhaps I'm missing the context into how this function is used server-side, but `WebsocketURL` is depended upon by https://github.com/mattermost/mattermost-redux/blob/ea17922990d7a8eed36af16facfc37d2223ac4a1/src/actions/websocket.js#L55. I think @crspeller also made some changes somewhere to support a different `WebsocketURL` than the `SiteURL`?

Reply:
Yup this should not be removed.

Reply:
How did this even happen... 🤔

Good catch.